scripts/asm-installer/tests/common.sh

```diff
@@ -843,7 +843,12 @@ create_source_instance_template() {
 
 create_custom_source_instance_template() {
   echo "Creating custom source instance template ${CUSTOM_SOURCE_INSTANCE_TEMPLATE_NAME}..."
-
+  exist_instance=$(gcloud compute instances list --filter="name=${CUSTOM_IMAGE_NAME}")
```

Comment:
Instead of cleaning up in each test run, can we clean up all the old instance template in https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/master/scripts/asm-installer/tests/setup_longterm_cluster#L34 which we are already doing?

Reply:
I don't have permission to do that

Reply:
resolved